src/app/shared/shared.module.ts

```diff
@@ -38,7 +35,10 @@ const COMPONENTS = [
 @NgModule({
   imports: [CommonModule, FontAwesomeModule, CustomMaterialModule, RouterModule, ReactiveFormsModule],
   declarations: [...COMPONENTS],
-  exports: [...COMPONENTS],
-  entryComponents: [AboutDialogComponent]
+  exports: [...COMPONENTS]
 })
-export class SharedModule {}
+export class SharedModule {
+  constructor(library: FaIconLibrary) {
+    library.addIcons(faGithub, faTwitter, faHeart, faHandsHelping, faBell, faBan, faInfo);
+  }
+}
```

Comment:
I think this change causes the icons to flicker. If you run `yarn dev:ssr` and you hard reload the page they are gigantic for a brief moment. That's not the case currently so we should see how to fix this in this PR.